server/src/main/resources/application.yaml

```diff
@@ -59,7 +59,7 @@ cors:
 
 voting:
   # 5 min
-  timeToVoteInSeconds: 300
+  timeToVoteInSeconds: 3000
```

Comment:
Can you return to 5 min please? Or it was done by intent?

Reply:
done